### ISSUE TITLE
Stabilise flaky marketplace e2e test

### ISSUE THREAD

**Issue by MTrost** [nuvla/ui#1095]
It tests if all apps shown in app store are really published by clicking on all apps and check the version number checkmark.

Probably better to check this by investigating the response payload for the apps we list in the app store (but I'm not sure if we return that flag) OR by checking that request payload has the correct filters set.

**Comment by 0xbase12** [comment]
yes the flag `published` is available in response resources and in request filter `published=true`
